app/assets/stylesheets/components/cart/_cart_link.scss

```diff
@@ -11,14 +11,16 @@
   background-position: bottom;
   position: relative;
 
+  //https://trello.com/c/CQKgoAUG/13-add-a-link-to-the-cart-to-the-header
```

Comment:
Why did you change existing CSS? Are you sure that this will not affect other places?

Reply:
It's okay. This css only applies to the linked cart. I only changed the backgroud color from blue to red.